src/Screens/Cart/index.tsx

```diff
@@ -3,89 +3,45 @@ import React from "react";
-                            <Icon name="rupee" size={20}/>
-                            <Text style={{fontWeight:'bold',fontSize:17,marginLeft:5}}>{item.item.item.price}</Text>
-                        </View>
-
-                        <View style={styles.quant}>
-                            <Icon name="minus" size={20} style={styles.quantIcon}/>
-                            <Text style={styles.quantText}>1</Text>
-                            <Icon name="plus" size={20} style={styles.quantIcon}/>
-                            <Text style={styles.quantText}>|</Text>
-                            <Text style={styles.quantText}>43</Text>
-                            <Icon name="chevron-down" size={20} style={styles.quantIcon}/>
-                        </View>
-
-                        
-                    </View>
-            </View>
-            
-
-            <View style={styles.cartCard}> 
-    
-                   <Image source={require('../../assets/woman/w6.jpg')} style={styles.cardImg}/>
-
-                    <View style={styles.cardText}>
+           <CartCard  image={item.item.item.img}
+                name={item.item.item.name}
+                info={item.item.item.info}
+                price={item.item.item.price}/>
 
-                        <Text style={{fontSize:25,fontWeight:'bold'}}>White Frock</Text>
-                        <Text style={{fontSize:16,paddingVertical:5}}>Floral umbrella dress</Text>
-                        <Icon name="close" size={20} style={styles.cardIcon}/>
-                        <View style={styles.Price}>
-                            <Icon name="rupee" size={20}/>
-                            <Text style={{fontWeight:'bold',fontSize:17,marginLeft:5}}>900</Text>
-                        </View>
-
-                        <View style={styles.quant}>
-                            <Icon name="minus" size={20} style={styles.quantIcon}/>
-                            <Text style={styles.quantText}>1</Text>
-                            <Icon name="plus" size={20} style={styles.quantIcon}/>
-                            <Text style={styles.quantText}>|</Text>
-                            <Text style={styles.quantText}>32</Text>
-                            <Icon name="chevron-down" size={20} style={styles.quantIcon}/>
-                        </View>
-
-                        
-                    </View>
-             
-            </View>
+           <CartCard  image={images.w6}
+                name={'White Frock'}
+                info={'Floral umbrella dress'}
+                price={'900'}/>   
             
             <View style={styles.totalCard}>
                <Text style={styles.totalCardText}>2 Items</Text>
                <Text style={styles.totalCardText}>{`Rs. ${item.item.item.price + 900}`}</Text>
             </View>
 
-            <PrimaryBtn name = {'Checkout'} navigation= {navigation}/>
+            <Buttonn name = {'Checkout'}  theme={'primary'} 
```

Comment:
Is there a specific reason to name this component **Buttonn** or is it just Typo?